pkgs/by-name/dc/dco-check/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "dco-check";
+  version = "0.5.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "christophebedard";
+    repo = "dco-check";
+    tag = version;
+    hash = "sha256-ux844Hc1D1CcW6T1M3OYW02NWfU6yyWCimFHFLI4J4o=";
+  };
+
+  build-system = with python3.pkgs; [ setuptools ];
+  dependencies = with python3.pkgs; [
```

```suggestion
  checkInputs = with python3.pkgs; [
```
These seem to be used only for tests